data/appr_model/channel.py

```diff
@@ -19,7 +19,7 @@ def get_channel_releases(repo, channel, models_ref):
             Channel.repository == repo,
             Channel.name == channel_name,
             Channel.tag_kind == tag_kind_id,
-            Channel.lifetime_end != None,
+            Channel.lifetime_end.is_null(False),
```

Comment:
Is that a change with made by Flake8?

Reply:
Isn't lifetime_end just an integer?

Reply:
flake8 complained about `!= None`, and @flavianmissi pointed that `.is_null` is the right way to do it.

https://github.com/quay/quay/blob/d9dbbd88dbb9a1da95ef46195bcfb5458db6a7b8/data/database.py#L1733
If that's the right model, lifetime_end can indeed be null.

Reply:
Does it complain on `is not None`?

Reply:
Ah, it's peewee's wrapper for  numbers.

Reply:
It doesn't complain on `is not None`, but `is not None` has different semantics and breaks tests.